app/lib/release_phase_notifier.rb

```diff
@@ -0,0 +1,15 @@
+class ReleasePhaseNotifier
+  def self.ping_slack
+    return unless ApplicationConfig["SLACK_WEBHOOK_URL"].present? && ApplicationConfig["SLACK_DEPLOY_CHANNEL"].present?
+
+    client = Slack::Notifier.new(
+      ApplicationConfig["SLACK_WEBHOOK_URL"],
+      channel: ApplicationConfig["SLACK_DEPLOY_CHANNEL"],
+      username: "Heroku",
+    )
+
+    client.ping("Release Phase Failed: #{ENV['FAILED_COMMAND']}")
+  rescue Slack::Notifier::APIError => e
+    Rails.logger.info("Slack API error occured: #{e.message}")
```

Comment:
`Rails.logger.error` ?

Reply:
Log level is debug in production and all we need is to see it in the little log file that the script outputs on Heroku so I figured this would do